examples/lifelong_learning/cityscapes/cityscapes-segmentation-lifelong-learning-tutorial.md

```diff
@@ -211,6 +211,23 @@ spec:
 EOF
 ```
 
+### GPU enabled (optional)
+If you want GPU to accelerate training or inference in Sedna, you can follow the steps below to enable GPU:
+
+> 1. Follow the instructions in  [nvidia-device-plugin](https://github.com/NVIDIA/k8s-device-plugin#quick-start) to make nvidia-docker to be docker runtime.
+> 2. Set config `devicePluginEnabled` to `true` and restart edgecore in the gpu edge node. Please refer to this [doc](https://kubeedge.io/docs/setup/config#configuration-edge-side-kubeedge-worker-node) for configuring edgecore.
```

Comment:
How should the edge container be configured when the runtime is containerd？

Reply:
Currently not tested on containerd for edge. And if it is supported by Nvidia for kubernetes, it will support for kubeedge and also Sedna.